src/stores/ReactionStore.js

```diff
@@ -38,7 +38,7 @@ class ReactionStore extends DataStore {
     * @method resolveID
     * @memberof ReactionStore
     * @instance
-    * @param {MessageReactionResolvable} role The role resolvable to resolve
+    * @param {MessageReactionResolvable} Reaction The MessageReaction to resolve
```

Comment:
Parameters should be in lowercase.
```suggestion
    * @param {MessageReactionResolvable} reaction The MessageReaction to resolve
```